extensions/amp-font/0.1/amp-font.js

```diff
@@ -0,0 +1,304 @@
+    } else {
+      this.onBeforeloadFont_();
+      this.loadFont_();
+    }
+  }
+
+
+  /**
+   * If the browser supports font loading events use them to load the font.
+   * @private
+   */
+  loadFontWithNativeApis_() {
+    /* style | variant | weight | size/line-height | family */
+    /* font: italic small-caps bolder 16px/3 cursive; */
+    let fontString = (
+      this.customFontStyle_ + " "
+      + this.customFontVariant_ + " "
+      + this.customFontWeight_ + " "
+      + this.customFontSize_ + " "
+      + this.customFontFamily_);
+    if (document.fonts.check(fontString)) {
+      this.onFontLoadSuccess_();
+    } else {
+      document.fonts.load(fontString).then(() => {
+        this.onFontLoadSuccess_();
+      });
+    }
+  }
+
+
+  /**
+   * Step 1 for loading font on browsers that don't support font loading events.
+   * Creates divs hidden from the viewport and measures dimensions for default
+   * fonts.
+   * @private
+   */
+  onBeforeloadFont_() {
+    var ncustomFontFamily_ = normalizeFontFamily_(this.customFontFamily_);
+    var containerElement = document.createElement('div');
+    containerElement.setAttribute(
+      'id', '-amp-font-container-' + ncustomFontFamily_);
+    style.setStyles(containerElement, {
+      fontSize: '48px',
+      fontVariant: 'normal',
+			left: '-999px',
+      lineHeight: 'normal',
+      margin: 0,
+      padding: 0,
+      position: 'absolute',
+			top: '-999px'
+    });
+    DEFAULT_FONTS_.forEach(font => {
+      let defaultFontElement = document.createElement('div');
+      let nfont = normalizeFontFamily_(font);
+      let defaultFontElementId = '-amp-font-'+ ncustomFontFamily_ + '-' + nfont;
+      defaultFontElement.setAttribute('id', defaultFontElementId);
+      defaultFontElement.textContent = TEST_STRING_;
+      style.setStyles(defaultFontElement, {
+        fontFamily: font,
+        margin: 0,
+        padding: 0,
+        whiteSpace: 'nowrap'
+      });
+      containerElement.appendChild(defaultFontElement);
+    });
+    document.body.appendChild(containerElement);
+    DEFAULT_FONTS_.forEach(font => {
+      let nfont = normalizeFontFamily_(font);
+      let defaultFontElementId = '-amp-font-'+ ncustomFontFamily_ + '-' + nfont;
+      let element = document.getElementById(defaultFontElementId);
+      this.measurements_[defaultFontElementId] = {
+        width: element./*OK*/offsetWidth,
+        height: element./*OK*/offsetHeight
+      };
+    });
+  }
+
+
+  /**
+   * Make the browser that don't support font loading events to download the
+   * custom font by creating an element (with text) not visible on the viewport.
+   * font download is detected by comparing the elements height and width with
+   * measurements taken in step1.
+   * @private
+   */
+  loadFont_() {
+    var haswidthChanged = false;
+    var hasHeightChanged = false;
+    var ncustomFontFamily_ = normalizeFontFamily_(this.customFontFamily_);
+    DEFAULT_FONTS_.some(font => {
```

Comment:
just want to make sure before i make the change, since:

<img width="1438" alt="screen shot 2015-10-27 at 9 00 38 pm" src="https://cloud.githubusercontent.com/assets/354746/10779428/da3aa634-7ced-11e5-9815-61373e66dd9b.png">